velox/docs/functions/spark/string.rst

```diff
@@ -178,6 +178,19 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT startswith('js SQL', 'SQL'); -- false
         SELECT startswith('js SQL', null); -- NULL
 
+.. spark:function:: str_to_map(string, entryDelim, keyValueDelim) -> map(string, string)
+
+    Returns a map by splitting ``string`` into entries with ``entryDelim`` and splitting
+    each entry into key/value with ``keyValueDelim``.
+    Only supports constant single-character ``entryDelim`` and ``keyValueDelim``. Throws
```

Comment:
nit: ``entryDelim`` and ``keyValueDelim`` must be constant single-character strings.